src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6207,7 +6207,9 @@ private BoundExpression MakeMemberAccessValue(BoundExpression expr, DiagnosticBa
                         lookupResult,
                         flags);
 
-                    if (!boundMethodGroup.HasErrors && boundMethodGroup.ResultKind == LookupResultKind.Empty && typeArgumentsSyntax.Any(SyntaxKind.OmittedTypeArgument))
+                    if (!boundMethodGroup.HasErrors && (boundMethodGroup.ResultKind == LookupResultKind.Empty ||
```

Comment:
>boundMethodGroup.ResultKind == LookupResultKind.Empty [](start = 56, length = 53)

Rather than adding something to this condition, could we simply remove it? BTW, what is the ```boundMethodGroup.ResultKind``` for our scenario?

Reply:
The `boundMethodGroup.ResultKind` for this scenario is `WrongArity`, huh. It seems that some check may actually be in place. Going to modify this to check for that condition instead to see if it works.